### PR TITLE
Multiple windows

### PR DESCRIPTION
**Pull request by nucleardreamer**
* Adds support for multiple `oak.load` calls, each call opens a separate window. 
* Client side now has `oak.id` available, which is that windows ID
* `oak.load` has an `id` property, and an optional `id` parameter

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Oak
-[![release](https://img.shields.io/badge/release-v1.1.1-green.svg)](https://github.com/OakLabsInc/oak/releases/tag/1.0.0)
+[![release](https://img.shields.io/badge/release-v1.2.0-green.svg)](https://github.com/OakLabsInc/oak/releases/tag/1.0.0)
```
Comment (aaron11496) [inline readme.md]: Should link to 1.2.0, not 1.0.0

File: build.sh [hunk 2]
```diff
@@ -23,7 +23,7 @@ if [[ $# -lt 2 && $1 == "app" ]]; then
         -v ${PWD##*/}-node-modules:/project/node_modules \
         -v ~/.electron:/root/.electron \
         electronuserland/electron-builder:wine \
-        /bin/bash -c "npm install && npm prune && npm run linux"
+        /bin/bash -c "npm install && npm prune && && npm run build && npm run dist"
```
Comment (aaron11496) [inline build.sh]: `&& &&` ??

**Review by aaron11496 — CHANGES_REQUESTED** [review]

File: lib/window.js [hunk 3]
```diff
@@ -60,15 +62,16 @@ class DisplayService extends EventEmitter2 {
       height: _this.opts.height,
       x: _this.opts.x,
       y: _this.opts.y,
+      hasShadow: false,
```
Comment (nirziv) [inline lib/window.js:65]: we have Boolean like kiosk without the has / is but now we have hasShadow. Should we be consistent

**Review by nirziv — APPROVED** [review]
just got one comment about the boolean names but not a check in stopper

File: build.sh (hunk 2 above)
Comment (0xVANISHED) [reply build.sh]: nice catch!

File: readme.md (hunk 1 above)
Comment (0xVANISHED) [reply readme.md]: also good catch!

**Review by aaron11496 — APPROVED** [review]

**Pull request merged**